system/emergency_handler/package.xml

```diff
@@ -11,6 +11,7 @@
 
```

Comment:
@mkuri Could you add you as a maintainer?

Reply:
Added by the following commit.
https://github.com/autowarefoundation/autoware.universe/pull/2070/commits/7b3dcb8812d72c1af683e4d31e4aaa503e8a6619